packages/e2e/pages/profile.page.ts

```diff
@@ -0,0 +1,21 @@
+import { By } from 'selenium-webdriver';
+import { TestRunContext } from '../support/test-run-context';
+import { envConfig } from '../env-keys';
+
+export default class HomePage {
+  public static navigateToProfile(): Promise<void> {
+    return TestRunContext.browser.get(`${envConfig.HOMEPAGE_URL}/profile`);
+  }
+
+  public static getPageTitle(): By {
+    return By.css(`h1`);
+  }
+
+  public static getForm(form: string): By {
```

Comment:
Could separate these out as 2 locators. In my opinion feels cleaner - but up to you!